packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -101,6 +101,7 @@ export const ChatComposite = (props: ChatCompositeProps): JSX.Element => {
    * @TODO Remove this function and pass the props directly when file-sharing is promoted to stable.
    * @private
    */
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
You can use `FileSharingOptions` return type. The type is defined unconditionally (the field `ChatCompositeOptions.fileSharing` is conditionally defined.

https://github.com/Azure/communication-ui-library/blob/e9622a0f52b0d3a3218d9507d6f2dc5f0d08d6ee/packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx#L77